en/mkdocs.yml

```diff
@@ -102,15 +102,17 @@ nav:
       - Develop Services:
           - Service Component Overview: develop-components/develop-services/service-component-overview.md
           - Develop a Service: develop-components/develop-services/develop-a-service.md
-          - Develop a service with Docker: develop-components/develop-services/develop-a-service-with-docker.md
+          - Develop a Service with Docker: develop-components/develop-services/develop-a-service-with-docker.md
           - Develop a Ballerina Service:  develop-components/develop-services/develop-a-ballerina-service.md
           - Expose a gRPC Endpoint via a Service: develop-components/develop-services/expose-a-grpc-endpoint-via-a-service.md
           - Expose a TCP Server via a Service: develop-components/develop-services/expose-a-tcp-server-via-a-service.md
           - Expose a UDP Server via a Service: develop-components/develop-services/expose-a-udp-server-via-a-service.md
           - Expose a Ballerina GraphQL Endpoint via a Service: develop-components/develop-services/expose-a-ballerina-graphql-endpoint-via-a-service.md
           - Expose a Prism-Based Mock Server Using an OpenAPI Specification: develop-components/develop-services/expose-a-prism-based-mock-server-using-an-openapi-specification.md
           - Expose a WebSocket Endpoint via a Service: develop-components/develop-services/expose-a-websocket-endpoint-via-a-service.md
-      - Develop an API Proxy: develop-components/develop-an-api-proxy.md
+      - Develop an API Proxy: 
```

Comment:
Shall we have the topics in this section as follows:

Develop an API Proxy
   - Develop From a Git Repository Source
   - Develop From Scratch or by Uploading an OpenAPI Specification
   
 WDYT?

Reply:
Isn't that a confusion when there is no any mention about the `API proxy` in the title ?

Reply:
Since these are going to be subtopics under "Develop an API Proxy" and also the context makes it clear that the topics are about developing an API proxy, the shorter titles can help avoid redundancy and keep the navigation concise.
Your point is valid if these titles appear in isolation or in search results without the navigation structure and context. So +1 to go with API Proxy in the title:

Develop an API Proxy
  - Develop an API Proxy from a GitHub Repository Source
  - Develop an API Proxy from Scratch or by Uploading an OpenAPI Specification